src/fidesops/schemas/saas/shared_schemas.py

```diff
@@ -31,3 +31,13 @@ class Config:
         """Using enum values"""
 
         use_enum_values = True
+
+class ConnectorParamRef(BaseModel):
```

Comment:
Moved `ConnectorParamRef` and `IdentityParamRef` here to avoid cyclic dependencies.